.alexrc

```diff
@@ -1,3 +1,3 @@
 {
-    "allow": ["invalid", "color", "colors", "white", "latin", "hook", "hooks"]
+    "allow": ["invalid", "color", "colors", "white", "latin", "hook", "hooks", "trap"]
```

Comment:
**note:** We were getting a warning about this, but I think the use of this word is OK in our case.